presto-main/src/main/java/io/prestosql/sql/SqlFormatterUtil.java

```diff
@@ -37,9 +40,11 @@ public static String getFormattedSql(Statement statement, SqlParser sqlParser)
             parsed = sqlParser.createStatement(sql, parsingOptions);
         }
         catch (ParsingException e) {
-            throw new PrestoException(GENERIC_INTERNAL_ERROR, "Formatted query does not parse: " + statement);
+            log.debug("Formatted query does not parse: \n\t\t statement: [%s] \n\t\t formatted: [%s]", statement, sql);
```

Comment:
Remove space before newline and after tab. Single tab should be sufficient:
```java
log.debug("Formatted query does not parse:\n\tstatement: [%s]\n\tformatted: [%s]", statement, sql);
```
I'm also thinking we don't need brackets around the statement, since it will be of the form `Query{...}` produced by `ToStringBuilder`:
```java
log.debug("Formatted query does not parse:\n\tstatement: %s\n\tformatted: [%s]", statement, sql);
```

Reply:
> Remove space before newline and after tab.

i kept it for readability (i know in final output it's not needed, but shouldn't hurt)

>  Single tab should be sufficient:

`at` lines are intended with a tab, that's why i used two tabs

> I'm also thinking we don't need brackets around the statement,

right

Reply:
It bugs me from a readability standpoint, since I read it thinking about how the final output looks like. But I understand your point.

> `at` lines are intended with a tab, that's why i used two tabs

Ah, makes sense.